prototype_v1/Student-View/utils/style.css

```diff
@@ -0,0 +1,92 @@
+.bttn-material-flat.bttn-default {
+    background: #fff;
+    color: black;
+}
+.skin-black .main-header .navbar {
+    background-color: #222d32;
+}
+.box.box-primary {
+    border-top-color: #222d32;
+}
+.bttn-material-flat {
+    margin: 0;
+    padding: 0;
+    border-color: black;
+    background: gray;
+    border-top-color: #222d32;
+}
+.main-header .logo {
+    font-weight: 700;
+}
+.skin-black .main-header>.logo {
+    background-color: #222d32;
+    color: #FFF;
+    border-bottom: 0 solid transparent;
+    border-right: 1px solid #222d32;
+    font-family: sans-serif;
+    font-weight: 350;
+}
+.skin-black .main-header .navbar>.sidebar-toggle {
+    color: #FFF;
+    border-right: 1px solid #f4f4f4;
+}
+.box.box-danger {
+    border-top-color: #FFF;
+}
+.bttn-unite.bttn-default {
+    border-color: #000000 !important;
+    color: #000000 !important;
+}
+.bttn-unite.bttn-default:before {
+    background: #dd4b39bf;
+    color: #000000 !important;
+}
+.bttn-unite.bttn-default:after {
+    background: #999999cf;
+    color: #000000 !important;
+}
+.bttn-unite.bttn-default {
+    border-color: #000000;
+    color: #000000 !important;
+}
+.bttn-unite.bttn-default:focus {
+    background:#d68e86 !important;
+}
+.skin-black .main-header>.logo:hover {
+    background-color: #f10909;
+}
+.vis-item.homework {
+    background-color: #ac1a2f;
+    color: white;
+    border-color:  #222d32;
+   
+}
+.vis-item.review {
+    background-color: #222d32;
+    color: white;
+    border-color: #ac1a2f;
+}
+.vis-item.vis-point.vis-selected, .vis-item.vis-selected {
+     opacity: 0.8;
+}
+
+a:link {
+  color: #ac1a2f;
+  text-decoration: underline;
+}
+
+a:visited {
+  color: #ac1a2f;
+  text-decoration: underline;
+}
+
+a:hover {
+  color: #ac1a2f;
+  text-decoration: underline;
+  font-weight: 700;
+}
+
+a:active {
+  color: #ac1a2f;
+  text-decoration: underline;
+}
```

Comment:
Change blue to davidson red?

![Screen Shot 2020-10-18 at 10 35 31 PM (2)](https://user-images.githubusercontent.com/52048606/96395526-d9171c80-1192-11eb-973e-bf3676909ec0.png)

Reply:
The blue as in the line on the tabs? Or the blue on the header / sidebar?

Reply:
got it right